### DISCUSSION TITLE
@designable/core里为啥会有对@formily/xxx的引用？

### DISCUSSION THREAD

**Discussion by christlee1989** [alibaba/designable#188] [Q&A] [answered]
我理解designable作为一个通用的基础搭建平台，formily只是使用它的其中一种表单实现，我们可以实现自己的类”formily“的实现。

![image](https://user-images.githubusercontent.com/10151385/147814527-69dac8d7-7e1c-40e8-97cc-9597974f640e.png)

**Comment by janryWang** [comment, marked as answer]
配置面板就是要依赖formily，标配，不要有疑问

**Reply by christlee1989** [reply]
> 

我还是不太理解，难道不是应该提供一个公共的schema，然后formily去extend然后自己扩展么?

**Reply by christlee1989** [reply]
还是说我应该把这个依赖，理解为就是designable的一部分？不要被它的名字所误导？
这里的依赖类似于 @formily/xxx === @designable/xxx ? 只是名字是formily开头的

**Reply by hou-pu** [reply]
直接fork过去，按需修改就行
